domain/secret/state/state.go

```diff
@@ -445,38 +466,37 @@ func (st State) updateSecret(
 	// so we may as well also include existing metadata as well so simplify
 	// the update statement needed.
 	existingSecretQuery := `
-WITH rev AS
-    (SELECT  MAX(revision) AS latest_revision
-    FROM     secret_revision
-    WHERE    secret_id = $secretID.id)
-SELECT
-     (sm.secret_id,
-     version,
-     description,
-     auto_prune,
-     rp.policy,
-     rev.latest_revision) AS (&secretInfo.*),
-     (so.owner_kind,
-     so.owner_id,
-     so.label) AS (&secretOwner.*)
-FROM secret_metadata sm, rev
+WITH rev AS (
+    SELECT MAX(revision) AS latest_revision
+    FROM   secret_revision
+    WHERE  secret_id = $secretID.id
+)
+SELECT (sm.secret_id,
```

Comment:
The original formatting with the select on its own line is visually much nicer as all the columns have consistent white space to their left